### PR TITLE
#207 - SfAddToCart

### PR DESCRIPTION
**Pull request by mayashavin**
# Related issue
[207](https://github.com/DivanteLtd/storefront-ui/issues/207)

# Scope of work
1. Add `SfAddToCart` with support for `slot`
2. Add unit test

# Screenshots of visual changes

Screen Shot 2019-08-23 at 17 28 25

With `slot` for quantity select input design:

Screen Shot 2019-08-23 at 17 28 46

With `slot` for Add to Cart button
Screen Shot 2019-08-23 at 17 28 38

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="sf-add-to-cart" data-test="sf-add-to-card">
+  <SfButton v-on="$listeners" class="sf-add-to-cart__button">
+    <!--@slot Custom content that will replace default Add to cart button design.-->
+    <slot name="add-to-cart-btn">
```
Comment (filrak) [inline packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.html]:
```suggestion
    <slot name="add-to-cart-btn-content">
```
Maybe something like this?

Comment (mayashavin) [reply packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.html]: right. Will change :)

File: packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import SfButton from "../../atoms/SfButton/SfButton.vue";
```
[the remark this answers is not in the document]
Comment (mayashavin) [reply packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.js]: got it

**Comment by filrak** [comment]
@mayashavin is this how it's supposed to look like? 
image

**Comment by mayashavin** [comment]
> @mayashavin is this how it's supposed to look like?
> image
@filrak It should look like this (without the arrow down)
Screen Shot 2019-08-23 at 17 28 25

**Comment by filrak** [comment]
https://deploy-preview-245--storefrontui-storybook.netlify.com/?path=/story/molecules-addtocart--basic it doesn't on preview. Are you sure you pushed the last changes?

**Comment by mayashavin** [comment]
Close this one. The correct PR is here: https://github.com/DivanteLtd/storefront-ui/pull/308

**Pull request closed without merging**